packages/adapter-cloudflare/index.js

```diff
@@ -3,24 +3,17 @@ import { fileURLToPath } from 'url';
 import { readFileSync, writeFileSync } from 'fs';
 import * as esbuild from 'esbuild';
 
-/**
- * @param {esbuild.BuildOptions} [options]
- */
+/** @type {import('.')} */
 export default function (options = {}) {
 	return {
 		name: '@sveltejs/adapter-cloudflare',
-		async adapt({ utils, config }) {
+		async adapt(utils) {
```

Comment:
It seems like it'd be more future-safe to keep this as `{ utils }` in case we want to add anything else in the future. I think we were trying to do that with all the public APIs. It's also slightly less for adapter authors to have to change to adapt to this PR

Reply:
the way i think of it is that `utils` already _is_ the future-proof bag of non-positional arguments. this...

```js
adapt(utils) {
  utils.rimraf(out);
  // ...
}
```

...is equivalent to this:

```js
adapt({ rimraf }) {
  rimraf(out);
  // ...
}
```

it's the same as endpoints taking a `request` as a single argument:

```js
export function get({ path, params }) {
  // ...
}
```

if we need to add more stuff in future, we could do so by putting it on `utils`.

this might feel 'nicer' if it wasn't called `utils` — not sure what a better name would be though.

Reply:
I don't really have any better ideas for names. Maybe `args`? That would feel more natural if `config` were added back in the future and would clarify that it's not just utils, but perhaps makes less sense in the short-term

Reply:
ended up with `builder`, per discord discussion, though remain open to alternatives